CHANGELOG.md

```diff
@@ -81,6 +83,12 @@ This document tracks pending changes to packages. It is facilitating the write-u
 
 - Style changes
 
+- Add support for hybrid (election & time-based) instruments
+
+- Create version consisting of more than hash of remaining claims
+
+- Create instruments even if they have a Zero claim
```

Comment:
Just for my understanding, could you remind me how it now works? We create the instrument but do not return any holding on it?

Reply:
This is in the lifecycle step, where we now create a new instruments even if it has a Zero claim:
https://github.com/digital-asset/daml-finance/blob/main/src/main/daml/Daml/Finance/Claims/Lifecycle/Rule.daml
I think is before the Holdings step (also happens if there are no holdings in a unit-style test).

Reply:
Yup, I was wondering if, as part of lifecycling, we return a holding on the expired instrument or not. If I remember correctly we should not be doing this